docs/getting-started/concepts/tags.md

```diff
@@ -0,0 +1,59 @@
+# Tags
+
+## Overview
+
+Tags is a way to filter Feast objects when listing the objects in the UI, Cli or when directly querying the registry.
```

Comment:
```suggestion
Tags in Feast allow for efficient filtering of Feast objects when listing them in the UI, CLI, or querying the registry directly. 
```

Reply:
@franciscojavierarceo - can we mention in our documentation saying "it provides similar functionality as  `kubernetes labels`" ?

Reply:
I guess we can, but I don't really see a reason to do that. imho, everyone know what a tag means anyway.

Reply:
Yeah, sure! Feel free to add.